config/crd/bases/eventing.keda.sh_clustercloudeventsources.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.15.0
```

Comment:
@kedacore/keda-contributors any ideas why this is all of a sudden necessary to pass the CI checks?

Reply:
v0.15.0 is already in the main, this PR should be probably rebased and then it will be okay:

https://github.com/kedacore/keda/blob/66d4c9595e917a86b390bbed6eed88b1b7cc5c92/config/crd/bases/eventing.keda.sh_cloudeventsources.yaml#L6

Reply:
thanks, it is indeed good now